src/main/resources/google_checks.xml

```diff
@@ -61,7 +61,7 @@
         <module name="RightCurly">
             <property name="id" value="RightCurlyAlone"/>
             <property name="option" value="alone"/>
-            <property name="tokens" value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT, INSTANCE_INIT"/>
+            <property name="tokens" value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT, INSTANCE_INIT, LAMBDA"/>
```

Comment:
We need confirmation on google style guides requirements for this from http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html or https://google.github.io/styleguide/javaguide.html .